### DISCUSSION TITLE
How to get the number of stock of items ?

### DISCUSSION THREAD

**Discussion by makaloni6** [saleweaver/python-amazon-sp-api#222] [Q&A]
Hello.

It may be a basic question.
I want to know the number of products in stock using ASIN.

I have tried some methods (get_item_offers, get_item, ...)
How do I get the inventory of a product with an ASIN?

**Comment by vikingcodes** [comment]
Amazon never provides a stock quantity of ASIN. 

Yes , if you want check quantity of your SKU and inventory then you can use inventory report.

**Reply by makaloni6** [reply]
thank you for replying!

But I want to know the stock of product.

**Comment by saleweaver** [comment]
Is this for FBA or MFN products?

**Reply by makaloni6** [reply]
Thank you for replying!

I want to know both.

**Reply by 0xs3n4** [reply]
How did you get the stock at ASIN level? Does it show the inventory of all sellers on the listing?

**Comment by 0xs3n4** [comment]
How did you get the stock at ASIN level? Does it show the inventory of all sellers on the listing?
